### PR TITLE
[13.0][MIG] bi_sql_editor

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration
Module migration stuck on #380

CC @ForgeFlow @legalsylvain @dominiquekonsuntack

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tests LGTM

**Comment by tonygalmiche** [comment]
I wanted to test the module on my personal computer.

However, module 'sql_request_abstract' is missing, which is not available in Odoo 13.

Otherwise, I think it will be worth reviewing the screenshots because the look is no longer the same with odoo 13.

Finally, a little suggestion: I think it would be good to add the list view

Thank you for this migration of this very interesting module

**Comment by AaronHForgeFlow** [comment]
@tonygalmiche Please see sql_request_abstract here https://github.com/OCA/server-tools/pull/1783

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+server-tools https://github.com/alfadil/server-tools.git 13.0-mig-sql_request_abstract
```
Comment (HviorForgeFlow) [inline oca_dependencies.txt]: @tonygalmiche you must merge this branch into server-tools OCA's one to have `sql_request_abrstract` available

Comment (tonygalmiche) [reply oca_dependencies.txt]: Thank you for the answer. It works perfectly. I did :
```
git clone https://github.com/alfadil/server-tools.git
cd server-tools/
git checkout 13.0-mig-sql_request_abstract
```

And I tested with this request : 

```
SELECT
  so.name as x_name,
  so.partner_id as x_partner_id,
  sol.product_id as x_product_id,
  sol.product_uom_qty as x_product_uom_qty,
  so.state as x_state
FROM sale_order so inner join sale_order_line sol on sol.order_id=so.id

```
Everything works fine. Thank you for this work

**Review by alan196 — APPROVED** [review]
Functional test 👍

**Review by tonygalmiche — APPROVED** [review]
Everything works fine. Thank you for this work

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
@HviorForgeFlow can you remove temporal dependencies commit? this is ready apart from that.

**Review by LoisRForgeFlow — APPROVED** [review]
Good now!

**Review by AdriaGForgeFlow — APPROVED** [review]

**Pull request merged**